source/retryable-writes/retryable-writes.rst

```diff
@@ -485,17 +495,20 @@ Consider the following pseudo-code:
      * propagate. */
     try {
       return executeCommand(server, retryableCommand);
-    } catch (NetworkException secondError) {
-      updateTopologyDescriptionForNetworkError(server, secondError);
-      throw secondError;
-    } catch (NotMasterException secondError) {
-      updateTopologyDescriptionForNotMasterError(server, secondError);
+    } catch (Exception secondError) {
+      if (secondError instanceof DriverException) {
```

Comment:
I would suggest the following to avoid nesting and also give more context to the DriverException being ignored (as we do above when server selection fails):

```
} catch (DriverException ignoredError) {
    throw originalError;
} catch (Exception secondError) {
    handleError(secondError);
    throw secondError;
}
```